src/components/MessageComp.jsx

```diff
@@ -0,0 +1,46 @@
+import { Box } from "@mui/material";
+import React from "react";
+
+const styling = {
+    "sender": {
+      "padding": "5px 8px 10px",
+      "background": "#D2E8C2",
+      "alignSelf": "flex-end",
+      "borderRadius": "10%",
+      "boxShadow": "#727272 0px 2px 2px",
+      "margin": "0 1px 4px 0 !important"
+    },
+    "receiver": {
+      "padding": "5px 8px 26px",
+      "background": "#F4F4F4",
+      "alignSelf": "flex-start",
+      "borderRadius": "10%",
+      "boxShadow": "#727272 0px 2px 2px",
+      "margin": "0 0 4px 1px !important"
+    },
+    "default": {
+      "padding": "5px 8px 26px",
+      "background": "#F4F4F4",
+      "alignSelf": "flex-start",
+      "borderRadius": "10%",
+      "boxShadow": "#727272 0px 2px 2px",
+      "margin": "0 0 4px 1px !important"
+    }
+}
+
+export default function MessageComp({ messageData }) {
+  const boxStyling = styling[messageData.messageType ?? "default"]
+  if (messageData.subcollectionData && messageData.subcollectionData.length > 0) {
+    return (
+      <Box sx={boxStyling}>
+        <div>
+          {/* <p>Sent by: {messageData.subcollectionData[0]?.sentby}</p> */}
+          <p>{messageData.subcollectionData[0]?.letter}</p>
+          <p>{JSON.stringify(messageData.subcollectionData[0]?.time)}</p>
```

Comment:
We'll need to present the creation/edit timestamp in a datetime format in a different font.

Reply:
Also, let's add the profile picture of the recipient.

Reply:
@charliefischer